furiousatoms/parsers/xyz_parser.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from furiousatoms.parsers.base_parser import BaseParser
+from furiousatoms.parsers.parser_util import float_or_zero
+
+class XYZParser(BaseParser):
+    def __init__(self) -> None:
+        super().__init__()
+        self.nextWordIndex = 0
+
+    def nextWord(self):
```

Comment:
All these class methods should be written as def next_word (following PEP8). Not nextWord.